samples/Azure.Management.Storage/Generated/BlobContainerOperations.cs

```diff
@@ -21,7 +21,7 @@ protected BlobContainerOperations()
         {
         }
 
-        public static readonly ResourceType ResourceType = "Azure.Management.Storage/BlobContainerOperations";
+        public static readonly ResourceType ResourceType = "Microsoft.Storage/storageAccounts/blobServices/default/containers";
```

Comment:
Can we add a test to verify the `ResourceType` value?

Reply:
we have unit tests here which is where that value is coming from https://github.com/Azure/autorest.csharp/blob/feature/v3/test/AutoRest.TestServer.Tests/Mgmt/ProviderSegmentDetectionTests.cs